src/tiles/Tiles.js

```diff
@@ -74,7 +74,7 @@ export const Tiles = ({
         </IconContainer>
       );
     } else {
-      return <TileContent>{content}</TileContent>;
+      return <TileContent>{tileName}</TileContent>;
```

Comment:
I believe this change will cause page to go blank upon clicking hyperlink!

Reply:
Yeah found just now!

Reply:
We should make it a practice to resolve any outstanding conversations before merging/closing a PR.